sdk/resourcemanager/azure-resourcemanager-compute/src/test/java/com/azure/resourcemanager/compute/VirtualMachineScaleSetOperationsTests.java

```diff
@@ -1456,6 +1457,107 @@ public void canDeleteVMSSInstance() throws Exception {
         computeManager.virtualMachineScaleSets().deleteInstances(rgName, vmssName, Collections.singleton(vmss.virtualMachines().list().stream().findFirst().get().instanceId()), false);
     }
 
+    @Test
+    public void canCreateFlexibleVMSS() throws Exception {
+        // create vmss with flexible orchestration type
+        VirtualMachineScaleSetInner options = new VirtualMachineScaleSetInner();
+        options.withOrchestrationMode(OrchestrationMode.FLEXIBLE)
+            .withPlatformFaultDomainCount(1)
+            .withLocation(region.name());
+
+        ResourceGroup resourceGroup = this.resourceManager.resourceGroups().define(rgName)
+            .withRegion(region.name())
+            .create();
+
+        Network network =
+            this
+                .networkManager
+                .networks()
+                .define("vmssvnet")
+                .withRegion(region.name())
+                .withExistingResourceGroup(resourceGroup)
+                .withAddressSpace("10.0.0.0/28")
+                .withSubnet("subnet1", "10.0.0.0/28")
+                .create();
+        final String vmssName = generateRandomResourceName("vmss", 10);
+        LoadBalancer publicLoadBalancer = createHttpLoadBalancers(region, resourceGroup, "1", LoadBalancerSkuType.STANDARD, PublicIPSkuType.STANDARD, true);
+
+        // create resource through raw method
```

Comment:
What is "create resource through raw method"?

Reply:
I modified the test and it's useless now, I'll delete it..

Reply:
The line is still there?

Reply:
My bad, deleted!